cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -618,13 +616,15 @@ you have a use case for it, please reach out to us at `cdap-user@googlegroups.co
 
        $ sudo yum install cdap cdap-gateway \
              cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
-             cdap-kafka cdap-master cdap-security cdap-web-app
+             cdap-kafka cdap-master cdap-security cdap-ui
 
    - Using APT (on one line)::
 
        $ sudo apt-get install cdap cdap-gateway \
              cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
-             cdap-kafka cdap-master cdap-security cdap-web-app
+             cdap-kafka cdap-master cdap-security cdap-ui
+
+       **Note:** We have deprecated the cdap-web-app package in favor of cdap-ui package 
```

Comment:
This line should be brought out to left to match the hyphen, as it applies to both Yum and APT. Also, it is not a literal command...